data-science-onramp/data-cleaning/clean.py

```diff
@@ -0,0 +1,179 @@
+import os
+import sys
+import re
+import datetime
+
+from py4j.protocol import Py4JJavaError
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import UserDefinedFunction, lit
+from pyspark.sql.types import StringType, IntegerType, FloatType
+
+
+PROJECT_ID = sys.argv[1]
+BUCKET_NAME = sys.argv[2]
+TABLE = f'{PROJECT_ID}.new_york_citibike_trips.RAW_DATA'
+
+def trip_duration(duration):
+    '''Convert trip duration to seconds.'''
+    if duration:
+        number = re.match('\d*.\d', duration)
+        if number:
+            number = float(number[0])
+        else:
+            return None
+
+        if 's' in duration:
+            return int(number)
+        elif 'm' in duration:
+            return int(number * 60)
+        elif 'h' in duration:
+            return int(number * 60 * 60)
+        else:
+            if number < 0:
+                return None
+            else:
+                return int(number)
+
+def station_name(name):
+    '''Replaces '/' with '&'.'''
+    if name:
+        return name.replace('/', '&')
+
+def user_type(user):
+    '''Converts user type to 'Subscriber' or 'Customer'.'''
+    if user:
+        if user.lower().startswith('sub'):
+            return 'Subscriber'
+        elif user.lower().startswith('cust'):
+            return 'Customer'
+        else:
+            return None
+
+def gender(s):
```

Comment:
"s" is a kind of vague variable name

Reply:
I agree. Think I copied it from setup.py but I will change it

Reply:
Haha, I will change it in setup.py as well!